Library/Homebrew/utils/analytics.rb

```diff
@@ -322,13 +322,14 @@ def formula_output(formula, args:)
 
         require "api"
 
-        json = Homebrew::API::Formula.fetch formula.name
+        timeout = args.analytics? ? nil : 2
```

Comment:
Can `2` perhaps be a constant?

Reply:
Maybe, but I had a quick glance at other uses of `:timeout` for `curl_output` and they just hard-code the timeout in the method call.

Reply:
```suggestion
        timeout = 2 unless args.analytics?
```
`nil` is implicit in Ruby here

I don't really understand this code, though: why do you want there to be no timeout when you're asking for analytics and a timeout set to `2` otherwise? Why 2?

Reply:
Thanks, applied. The idea is that `brew info foo` should be as fast as possible, and, unless you called `brew info` with `--analytics`, the analytics data will likely be what you're least interested in, so it doesn't make sense to wait indefinitely for it.